addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig

```diff
@@ -20,6 +20,11 @@
                 <div class="col-md-3 text-left justify-content-center align-self-center">
                     <div class="row">
                         <div class="col-md-12">
+                            <p class="theme-price-old small mb-0"><del>
+                                    {{ adv.standard_price.currency(null,'currency') != adv.price.currency(null,'currency')
```

Comment:
What if the standard price was 0, did you check?

Reply:
https://github.com/openclassify/openclassify/pull/678/commits/0a7b7386a7ca3e23c5b9cbaa4223548eb3a626c2